### ISSUE TITLE
OC 8 Serverside Sharing

### ISSUE THREAD

**Issue by apeltzer** [owncloud/core#14571]
Hi!

I'm running two instances of OC8, both with self signed SSL certificates (which is fine for my purpose). I created a folder on Host I and wanted to share this via Server-side-Sharing with Host II, which was pretty straigthforward. However, when I tried to add the link to my private owncloud, I get asked whether I want to add the URL (sure!) to my owncloud. Clicking on confirm, then results in a error popup: 

"Invalid or not trusthworthy SSL certificate."

Well awesome, but I'm fine with that and don't want to buy one ;-)

Is there a way to solve this except for buying a certificate?

**Comment by LukasReschke** [comment]
Ref https://github.com/owncloud/core/issues/13175

In your ownCloud 8 personal settings you might be able ti import a certificate in the meantime. Let us know how that worked out (https://github.com/owncloud/core/pull/10420)

**Comment by apeltzer** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Hi,

this is what I just did and I still get the same error message. Export
via Chrome/Firefox, Import via user settings - listed in my list of
trusted servers but nothing happens except for the same error message.



On 02/27/15 13:32, Lukas Reschke wrote:
> Ref https://github.com/owncloud/core/issues/13175
> 
> In your ownCloud 8 personal settings you might be able ti import a
> certificate in the meantime. Let us know how that worked out
> (https://github.com/owncloud/core/pull/10420)
> 
> --- Reply to this email directly or view it on GitHub: 
> https://github.com/owncloud/core/issues/14571#issuecomment-76387813
>
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQIcBAEBCAAGBQJU8IzEAAoJEKSpMitQr5WgBnkP/jc26538yAlU++qF2Ed2ZEj/
fyLHT5PJ84uNgpz2bhu20mpDhl5n4YWcoVhCVAYdnZOq2rvez+4wIvgKsaHrLgnX
DmO8sW5dR5XPZNqTiSRgPOtnkc4XuCGPGb2nA6UKPniVROpbWzqVWy6HcyLDBTus
7HZsCB8D81RHR3VgvVmpv3sxSKPjqYGomC9r53VfsUB5/LqDmk1wjYRxaIPjkva4
F394X7AHpMfToCoHQsNkiuTKzKeJeu7Euifh1GEUkkwY5JTqzLhDP2g0To9CWIlR
7Qx1+XXfz7TvsDcR1aB3hocjI0LhPB57Xl9fSU0AwvOsX3C2WwAGNziS3//ZAC18
GddhwLaTGTbL3qFYo5nIpti70DkuUImLCKUubQueXn9CABNiUOdhHg87xjwyzPMk
as1I5zOKQl+gqm27QNkxYiXje++gyBDPXhA/x8U+Ul0UiaKWEaczT+ijH35ufHFo
Kub8cgzpUcCRgCztuWpuQjPhONAKwaDWPZSX+XXlUVQ69UguGCmqpj0o3xpp6OGa
I0uN93p17b5xo1nZyWDcG2NKxPSKJ07J9Hqb3hQcWXcXAQs8t3/gFDXqb10sxX59
KVdjI2R0DG1XNPbwj2ceht+vwubF312CJhez4nWkXPJ4mFHn8sb6XgGIeopD+VLK
VBhTK3jXqLkOPbNLiuiA
=lscz
-----END PGP SIGNATURE-----

**Comment by Siedlerchr** [comment]
I just encountered the same problem, but my configuration is a bit different:
My owncloud (8.0.2) instance A is fully signed by Comodo CA and I tried to import a share from another owncloud instance B, which is self signed.

Add share from B to A doesn't work.
I got the same error as above, complaining about invalid or non trusworthy certificate
Then I exported the self signed CA from Firefox and imported it in my owncloud instance.
Same error message. In my log there is also no entry regarding any problems.
